kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/BigQuerySinkTask.java

```diff
@@ -159,10 +145,10 @@ public void put(Collection<SinkRecord> records) {
 
         if (!tableWriterBuilders.containsKey(table)) {
           TableWriter.Builder tableWriterBuilder =
-              new TableWriter.Builder(bigQueryWriter, table, record.topic());
+              new TableWriter.Builder(bigQueryWriter, table, record.topic(), recordConverter);
```

Comment:
I feel like the record converter should be applied to the record to get it into the format that .addRecord() takes. Does the TableWriter really need the record converter?